packages/webpack-loaders/src/gen-css-bundles.js

```diff
@@ -0,0 +1,95 @@
+import { assign } from "lodash";
+import * as t from "babel-types";
+
+import { fromObject } from "interlock/lib/util/ast";
```

Comment:
Would it be worthwhile to split out AST utils into a separate mini-library, do you think?  One upside of doing it this way is that the loader should reference the same interlock version that is consuming the loader itself.